### ISSUE TITLE
Is this still being developed?

### ISSUE THREAD

**Issue by WillTheDeveloper** [wonoly/search#2]
I noticed its in Laravel and I am quite experienced with it. I was wondering if it was still being developed and there was any roadmap or list of things that need to be added?

**Comment by mauro-balades** [comment]
yes. It is being developed. I just took a break and worked on other projects.

**Comment by WillTheDeveloper** [comment]
Ah okayy
